pipeline/pipeline.py

```diff
@@ -0,0 +1,60 @@
+"""
+Pipeline script that connects the ETL scripts,
+Runs the pipeline in a loop every 1 min.
+"""
+
+import time
+from os import environ
+
+from dotenv import load_dotenv
+
+from extract import fetch_all_plant_data, convert_plant_data_to_csv
+from transform import standardize_country_name, remove_rows_with_null, check_soil_moisture_valid, check_soil_temp_valid, normalize_datetimes, change_temp_and_moisture_to_two_dp
+from load import create_database_connection, insert_into_recordings_table
+
+if __name__ == "__main__":
```

Comment:
Like Shivani said, might be good to split up this into functions - maybe have extract, transform and load functions?